src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useState, useEffect } from "react";
+
+
+
+export const LoginView = ({onLoggedIn}) => {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+   
+ 
+    const handleSubmit = (event) => {
+
+        event.preventDefault();
+
+        const data = {
+            Username: username,
+            Password: password
+        };
+        fetch("https://movie-api-7p14.onrender.com/account/login", {
```

Comment:
I get this error <pre>Cannot POST /account/login</pre> when I try to login, do you have that endpoint on your backend API?

Always make sure you're using the right endpoint, verify that on postman.